### PR TITLE
Metrics Tags Updates

### LINKED ISSUE #227: Feature Request: Add More Metrics Tags
Add `topic` tag to list of included tags for kafka metrics.

For consumers, also add the `group-id` tag.

https://kafka.apache.org/21/javadoc/?org/apache/kafka/common/MetricName.html

### PR DESCRIPTION
**Pull request by brianwyka**
- Add `topic` as an included tag for metrics by default.
- Add `@Override` for consumer metrics to also include `group-id` tag

Closes #227

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-projectVersion=2.1.1.BUILD-SNAPSHOT
+projectVersion=2.2.0
```
Comment (graemerocher) [inline gradle.properties:1]: Please roll back this change

**Review by graemerocher — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 1 above)
Comment (brianwyka) [reply gradle.properties]: @graemerocher, rolled back.

**Comment by graemerocher** [comment]
Needs a test as well

**Comment by brianwyka** [comment]
> Needs a test as well

@graemerocher, happy to add some tests.  Is there a spec I can update, or will this be a new test?  Didn't see anything in there already.

**Comment by alvarosanchez** [comment]
If you didn't see any fit, just create a new spec.

**Comment by ctoestreich** [comment]
@brianwyka The tests I wrote to hit the metrics endpoint for the kafka metrics could be useful here as you can hit the endpoint and check the tags in the response?  You shoud be able to marry together a test container spec and then after messages are sent and received check the metrics endpoint for metrics tags?

**Comment by brianwyka** [comment]
I will be revisiting this to add tests today and tomorrow @alvarosanchez

**Comment by brianwyka** [comment]
Attempting to make a change in kafka clients as well:
https://github.com/apache/kafka/pull/9584

**Pull request closed without merging**